.github/workflows/testdocker.yml

```
@@ -0,0 +1,12 @@
+name: test docker auth
+on:
+    workflow_dispatch:
+jobs:
+    build-and-push-image:
+        runs-on: ubuntu-latest        
+        steps:
+            - name: Login to Docker Hub
+              uses: meta-introspector/login-action@v3
+              with:
+                username: ${{ vars.DOCKER_HUB_USERNAME }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)